src/platform/nxp/k32w/k32w1/DefaultTestEventTriggerDelegate.h

```diff
@@ -25,12 +25,9 @@ namespace chip {
 class DefaultTestEventTriggerDelegate : public TestEventTriggerDelegate
```

Comment:
Are these platform specific classes doing anything different now than your SimpleTestEventTriggerDelegate?

Also, kQueryTrigger - that seems to have disappeared - was that not used anywhere?

Reply:
- They call `emberAfHandleEventTrigger` which is how those platform examples are hooked-up. The init of the TestEventTriggerDelegate is not in the examples, but rather in the platform main, so I cannot add more handlers and that's how they did the hookups.
- kQueryTrigger was not doing anything, and not in allowable range. Will ask NXP what it did, but certainly nothing in SDK.